docs/arcaflow/workflows/expressions.md

```diff
@@ -419,7 +420,7 @@ Unary operations are operations that have one input. The operator is applied to
 
 | Operator | Description                               |
 |----------|-------------------------------------------|
-| -        | [Negation](#Negation)                     |
+| -        | [Negation](#negation)                     |
```

Comment:
Does the lowercase reference to a capitalized anchor work? (Yeah, if I was in rich diffs view, maybe I could tell. 😦 )

Wow: OK, do *not* click on that link. It's a weird extradimensional gateway, and you'll regret it. I warned you. So does it actually work in preview mode in the IDE? Will it work on the real site?

Reply:
i would suggest [rendering the docs locally](https://github.com/arcalot/arcalot.github.io?tab=readme-ov-file#adding-content) to see the content you added